app/components/timeline_entry/component.html.erb

```diff
@@ -61,6 +61,25 @@
         has changed from <strong class="govuk-!-font-weight-bold"><%= description_vars[:old_value] %></strong>
         to <strong class="govuk-!-font-weight-bold"><%= description_vars[:new_value] %></strong>.
       </p>
+
+    <% elsif timeline_event.application_declined? %>
+      <p class="govuk-body">
+        Status has changed to: <%= render(StatusTag::Component.new(:declined)) %>
+      </p>
+      <p>
+        <% TeacherInterface::ApplicationFormViewObject.new(application_form: timeline_event.application_form).declined_reasons.each do |title, reasons| %>
+          <% if title.present? %>
+            <h4 class="govuk-heading-s"><%= title %></h4>
+          <% end %>
+
+          <ul class="govuk-list govuk-list--bullet">
+            <% reasons.each do |reason| %>
+              <li><%= simple_format reason %></li>
+            <% end %>
+          </ul>
+        <% end %>
+      </p>
```

Comment:
I don't think this section needs to be wrapped in a paragraph (`<p>`), since we've got headings and lists inside which doesn't quite make sense.